Sources/Lumina/UI/Extensions/ViewControllerButtonFunctions.swift

```diff
@@ -77,7 +77,6 @@ extension LuminaViewController {
         LuminaLogger.notice(message: "torch mode should be set to on")
         camera.torchState = .on(intensity: 1.0)
         self.torchButton.updateTorchIcon(to: SystemButtonType.FlashState.on)
-        //swiftlint:disable empty_enum_arguments
       case .on(_):
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/45e47f954408babfc22eace2d454b922cb0007fe/Sources/Lumina/UI/Extensions/ViewControllerButtonFunctions.swift#L80"></span>Arguments can be omitted when matching enums with associated values if they are not used.<br />
<code>empty_enum_arguments</code> <code>ViewControllerButtonFunctions.swift:80</code></td>
    </tr>
  </tbody>
</table>